src/main.rs

```diff
@@ -48,3 +50,19 @@ fn make_ui() -> impl Widget<AppState> {
     );
     Controller::new(col)
 }
+
+/// If there was an argument passed at the command line, try to open it as a .ufo
+/// file, otherwise return blank state.
+fn get_initial_state() -> AppState {
+    let mut state = AppState::default();
+    if let Some(arg) = std::env::args().skip(1).next() {
```

Comment:
No structopt? :)

Reply:
Not until I accept at least _two_ distinct arguments. ;)